src/scribe_data/cli/query.py

```diff
@@ -0,0 +1,122 @@
+import json
+import csv
+from pathlib import Path
+from typing import Optional
+from .utils import LANGUAGE_METADATA, LANGUAGE_MAP
+
+DATA_DIR = Path('scribe_data_json_export')
+
+def query_data(language: str = None, word_type: str = None, output_dir: Optional[str] = None, overwrite: bool = False, output_type: Optional[str] = None) -> None:
+    if not (language and word_type):
+        print("Error: You must provide both --language (-l) and --word-type (-wt) options.")
```

Comment:
For places like this, let's `raise ValueError(...)` :)

Reply:
This way we don't need to `return` afterwards as well, and the error output will be similar to what people expect with the full trace rather than just a print statement.